en/team.md

```diff
@@ -0,0 +1,32 @@
+---
+title: About us
+lang: en
+ref: team
+layout: default
+menu_position: 2
+permalink: /en/team
+redirect_from:
+  - /team
+  - /en/team.htm
+  - /en/49-content.htm
+---
+
+## About us
+
+The Digital Transformation Team was born to build the "operating system" of the
+country, a series of fundamental components on top of which we can build simpler
+and more efficient services for the citizens, the Public Administration and
+businesses, through innovative digital products.
+
+The Commissioner's Team has been established on September 16, 2016 with an
+initial two-year mandate, up to September 16, 2018.   
+The <a href="http://presidenza.governo.it/AmministrazioneTrasparente/DisposizioniGenerali/AttiGenerali/DpcmOrganismiCollegiali/DPCM_20160916_CommStraord_AgendaDigitale.pdf" title="Prime-Ministerial Decree of Digital agenda commissarial structure" target="_blank">
```

Comment:
non sono sicuro che "Prime-Ministerial" abbia senso, probabilmente dovrebbe essere "Prime Minister Decree for the commissarial structure of the Digital Agenda" ?

Reply:
A [Ministerial Order](https://en.wikipedia.org/wiki/Ministerial_order) is a thing for sure.